app/src/main/java/pl/edu/agh/emotionalrobot/MainActivity.java

```diff
@@ -33,10 +53,30 @@ public void onClick(View v) {
                     //TODO put an animation on top of everything
                     emotionDataGatherer.startGatheringEmotions(updateSender);
                 } catch (Exception e) {
-                    Toast.makeText(getApplicationContext(), "Fill the interval", Toast.LENGTH_SHORT);
+                    Toast.makeText(getApplicationContext(), "Fill the interval", Toast.LENGTH_SHORT).show();
                 }
             }
         });
 
     }
+
+    private VideoEmotionRecognizer initializeVideoEmotionRecognizer() throws IOException {
+        CameraManager manager = (CameraManager) getSystemService(Context.CAMERA_SERVICE);
+        final int rotation = getWindowManager().getDefaultDisplay().getRotation();
+        VideoEmotionRecognizer recognizer = new VideoEmotionRecognizer(getApplicationContext(), rotation, manager, this);
```

Comment:
return new VideoEmotionRecognizer(getApplicationContext(), rotation, manager, this);